homeassistant/components/atag/climate.py

```diff
@@ -44,7 +44,9 @@ class AtagThermostat(AtagEntity, ClimateEntity):
     _attr_hvac_modes = HVAC_MODES
     _attr_preset_modes = list(PRESET_MAP.keys())
     _attr_supported_features = (
-        ClimateEntityFeature.TARGET_TEMPERATURE | ClimateEntityFeature.PRESET_MODE
+        ClimateEntityFeature.TARGET_TEMPERATURE
+        | ClimateEntityFeature.PRESET_MODE
+        | ClimateEntityFeature.TURN_ON
```

Comment:
I don't think this makes sense here, there is no "off" state at all, so there is also no turning on?

Reply:
In that case, I will advocate the full revert of all of this. If the flag is being misused, there is not reason to implement it in the first place.

../Frenck

Reply:
I and @frenck had a chat about this on discord, we agreed we don't see a need to revert all the changes right now.

- It's a good step forward to allow climate entities to opt in or out from turn_on / turn_off, and precisely because that wasn't the case before there's going to be all sort of weird things when we look closely at existing integrations.
- We should not just set the turn_on / turn_off flags to do the same as the automatic logic, but actually think about what it means. In this case, as well as in the Nibe case in https://github.com/home-assistant/core/pull/109140 we may not want to set the `TURN_ON` flag
- We either need to adjust the automatic logic or a way for integrations to opt out of the automatic logic, if Atag and Nibe don't support the `turn_on` service, the automatic logic should not flag support for that.

Reply:
It has been addressed in the `climate` component so it will be updated accordingly.